### DISCUSSION TITLE
EDV TV [CO]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#397] [Show and tell]
### Channel ID

EDVTV.co

### Stream URL

https://tv.portalexpress.es:3913/hybrid/play.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://tv.portalexpress.es:3913/hybrid/play.m3u8
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/7253922/232255354-fc0f79f4-003e-43f9-8763-dd87040559f1.png">

**Comment by freearhey** [comment]
https://tv.portalexpress.es:3913/hybrid/play.m3u8 (offline)
<img width="367" alt="image" src="https://user-images.githubusercontent.com/7253922/233810611-09d5ce6f-42dd-47d6-90fa-9d698101566d.png">
